api/datasets.go

```diff
@@ -24,6 +24,7 @@ import (
 // DatasetHandlers wraps a requests struct to interface with http.HandlerFunc
 type DatasetHandlers struct {
 	lib.DatasetMethods
+	inst     *lib.Instance
 	remote   *lib.RemoteMethods
 	node     *p2p.QriNode
 	repo     repo.Repo
```

Comment:
right now `DatasetHandlers` needs to work with "Dispatch-ified" and non "Dispatch-ified" `DatasetMethods`. When each is converted, we can remove the `lib.DatasetMethods`.